web/css/review.css

```diff
@@ -8,6 +8,27 @@ code {
   font-family:Courier, 'Courier New', Monaco, monospace;
 }
 
+.f008 {
+  font-family:Courier, 'Courier New', Monaco, monospace;
+  border: 1px solid #000;
+  padding: 3px;
+  margin: 4px;
+  letter-spacing: .2rem;
+  text-align: center;
+  display: inline-flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.f008 .f008-annotation {
+  margin-bottom: 0.1em;
```

Comment:
using `bottom` and `left` are totally fine and work! i've been trying to use [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_logical_properties_and_values) instead (for eventual i18n efforts) when I add/update styles:

`margin-bottom` => `margin-block-end`
`text-align: left` => `text-align: start`

not a necessary change, just thought you'd like to know!

the styling all looks good 👍

Reply:
There were a couple more to do in the main CSS file, I'll leave a couple of stragglers elsewhere in the repo on the to-do list for an eventual CSS cleanup.